libs/designsystem/icon/src/icon-registry.service.spec.ts

```diff
@@ -59,15 +64,18 @@ describe('KirbyIconRegistryService', () => {
       consoleWarnSpy = spyOn(console, 'warn');
     });
 
-    it('should add the icon to the regitry', () => {
+    it('should return icon added to the registry', () => {
       service.addIcon('name1', 'svg1');
-      expect(service.getIcons()).toEqual([{ name: 'name1', svg: 'svg1' }]);
+      const expectedIcon = { name: 'name1', svg: 'svg1' };
+
+      expect(service.getIcon('name1')).toEqual(expectedIcon);
     });
 
     it('should only add distinct icon names', () => {
       service.addIcon('name1', 'svg1');
       service.addIcon('name1', 'svg2');
-      expect(service.getIcons()).toEqual([{ name: 'name1', svg: 'svg1' }]);
+      const expectedIcons = [...defaultIcons, { name: 'name1', svg: 'svg1' }];
```

Comment:
```suggestion
      const expectedIcons = [...defaultIcons, icon1];
```

Any reason to not use `icon1` here?